docs/docs/reference/contextual/extension-methods.md

```diff
@@ -103,7 +103,7 @@ a single extension and enclose all methods in braces or an indented region follo
 Example:
```

Comment:
The text above this line would also need to be adjusted.

Reply:
Well spotted. I'll update the doc.